test/core/gprpp/stat_test.cc

```diff
@@ -31,6 +31,9 @@
 #include "src/core/lib/iomgr/load_file.h"
 #include "test/core/util/test_config.h"
 
+#define SERVER_KEY_CERT_DIR "src/core/tsi/test_creds"
```

Comment:
This contains forward slashes.  How is that going to work on Windows?

Reply:
I might be wrong, but I was under the impression that these tests using credential paths were only run in non-Windows platforms. For example, [this test](https://github.com/grpc/grpc/blob/master/test/core/security/BUILD#L250-L260) also uses the forward slash credential paths, but without tags of "no windows". Did I understand this correctly?

Reply:
Any test not marked as "no_windows" will run on Windows.

Maybe there's something that allows forward-slashes to work on Windows?  @nicolasnoble, is that expected to work?

Reply:
@nicolasnoble
Hi Nicolas, any ideas on why these paths using forward slashes work on Windows? I thought it could be due to the macros defined in https://github.com/grpc/grpc/blob/master/include/grpc/impl/codegen/port_platform.h, but was not entirely sure of it. Could you please confirm that? Thanks!

Reply:
I chatted about this with Nico, and he said that apparently forward slashes do actually work on Widows, but they're not really supposed to, so we shouldn't depend on it.

(There's apparently some nice support in C++17 for doing this in a cross-platform way, but we are still stuck on C++11, so that doesn't really help us.)

How about this: Let's make this API take its input as `std::vector<absl::string_view>`.  That way, we can join as many path components as we want in a single call.  And we can represent hard-coded paths like this as an array:

```
const char* kServerCertPath[] = { "src", "core", "tsi", "test_creds" };
```

Then we can convert it to a path string like this:

```
std::vector<absl::string_view>(kServerCertPath, kServerCertPath + GPR_ARRAY_SIZE(kServerCertPath));
```

And we can pass that into this API to get a string.  WDYT?

Reply:
Yes, sure! I will do it. Just to make sure, our file watcher provider won't need this, because it now takes three paths directly. Do we still want to make this change to avoid relying on the C++17 feature?

Reply:
That's a good point.  If we have no pressing need for this right now, I'm fine with closing this PR.

Reply:
OK sounds good! I will close this for now. Feel free to reopen if this is in need in the future.